react/components/BuyButton/index.js

```diff
@@ -45,44 +42,60 @@ export class BuyButton extends Component {
   }
 
   handleAddToCart = async () => {
-    const { skuItems, isOneClickBuy, orderFormContext, push } = this.props
+    const { addToCart, skuItems, isOneClickBuy } = this.props
     this.setState({ isAddingToCart: true })
 
-    const variables = {
-      items: skuItems.map(skuItem => {
-        const { skuId } = skuItem
+    if (isOneClickBuy) location.assign(CONSTANTS.CHECKOUT_URL)
+
+    let toastMessage = null
```

Comment:
```suggestion
    let updateToastMessage;
```

Reply:
But it doesn't really `updates` the message. It shows the message 🤔

Reply:
Maybe `showToastMessage`. If some function will perform a side-effect, it is expected that its name indicate that. Anyone would guess that `toastMessage` would hold a string value later, seeing this line, don't you think?

Reply:
Done